### PR TITLE
Switching to phpbench/phpbench based benchmarks

### PR DESCRIPTION
**Pull request by webimpress**
/cc @Ocramius

### CODE REVIEW

**Comment by froschdesign** [comment]
@webimpress 
Can you check Travis please:

https://travis-ci.org/zendframework/zend-stdlib/jobs/285547904#L558

**Comment by webimpress** [comment]
@froschdesign We need #74 merged before. Travis CI configuration is not the best at this moment, and it force install (`--ignore-platform-reqs`) dependencies which are not compatible with PHP 7.0 (for example `doctrine/annotations 1.5.0` requires PHP 7.1)...

**Comment by froschdesign** [comment]
@webimpress 
Right, in the meantime I have also seen this. I will update the Travis file and remove `allow_failures`, because the problem is already fixed (3921c941891a05189686886dbc4dadfbba089156)

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,8 @@ matrix:
     - php: 7
       env:
         - DEPS=locked
-        - CHECK_CS=true
+        - CS_CHECK=true
```
Comment (webimpress) [inline .travis.yml]: The variable was wrong!! Please see line 71/73.

Yeah, I know... the best if we have it in separate PR...

**Comment by webimpress** [comment]
@froschdesign something definitely is wrong with this build, please note that I've moved cs-check, coverage and benchmarks to PHP 7.1 but build https://travis-ci.org/zendframework/zend-stdlib/builds/292168388?utm_source=github_status&utm_medium=notification still runs them on PHP 7! :-(

**Comment by webimpress** [comment]
@froschdesign rebased one more time, maybe now will be 💚

**Comment by froschdesign** [comment]
@webimpress
Benchmark is running: https://travis-ci.org/zendframework/zend-stdlib/jobs/292185106#L640

**Comment by webimpress** [comment]
Ok, so benchmarks are ok, the problem is with cs-checks, because the variable was wrong and we haven't run it at all. I'd suggest to merge #68 and fix these cs issues there. What do you think @froschdesign ?

**Comment by webimpress** [comment]
@froschdesign Rebased, I hope last time !

**Comment by webimpress** [comment]
💚

**Comment by froschdesign** [comment]
@webimpress 
Thanks! 👍 👍 👍

**Pull request merged**